src/walletProvider.ts

```diff
@@ -144,22 +160,22 @@ export class WalletProvider {
     }
 
     private getTxSignReturnValue(urlParams: any): PlainSignedTransaction[] {
-        console.info(`Received urlParams: ${urlParams}`);
+        console.info("getTxSignReturnValue(), urlParams:", urlParams);
```

Comment:
the console info is still needed?

Reply:
Kept for debugging (were useful in the past).